lib/irq.c

```diff
@@ -180,19 +180,32 @@ irqs_set_state(vfu_ctx_t *vfu_ctx, struct vfio_irq_set *irq_set)
     cb(vfu_ctx, irq_set->start, irq_set->count, mask);
 }
 
+static inline int*
```

Comment:
"int *" please. Also no need for "inline" keyword.

Reply:
ok